tests/network/localnet/liblocalnet.py

```diff
@@ -155,3 +161,74 @@ def client_server_active_connection(
             server_port=port,
         ) as client:
             yield client, server
+
+
+@contextlib.contextmanager
+def create_nncp_localnet_on_secondary_node_nic(
+    worker_node: Node, nodes_available_nics: dict[str, list[str]], mtu: int | None = None
+) -> Generator[libnncp.NodeNetworkConfigurationPolicy, None, None]:
+    """Create NNCP to configure an OVS bridge on a secondary NIC across all worker nodes.
+
+    Note:
+        This function assumes homogeneous hardware—all workers must have a NIC with
+        the same name as the one selected from worker_node. The configuration is applied to
+        all workers to support anti-affinity scheduled VMs.
+
+    Args:
+        worker_node: Node used to determine which NIC to configure.
+        nodes_available_nics: Dict mapping node names to their available NICs.
+        mtu: Optional MTU to configure on both the physical NIC and bridge.
+
+    Yields:
+        The created NodeNetworkConfigurationPolicy.
+    """
+    bridge_name = "localnet-ovs-br"
```

Comment:
I would append a randomness to it, protecting the tests from leaks.
See how we have done the same for VMs.